shoop/core/models/addresses.py

```diff
@@ -54,51 +55,10 @@ class Labels:
         ENABLED = _('enabled')
 
 
-class AddressManager(models.Manager):
```

Comment:
Now that @tulimaki squashed the address splitting changes to a single commit, it's fine IMHO.

@tulimaki Just add the entry to ChangeLog.rst about this change within the "Split address into mutable and immutable address" commit, please.